app/utils/hash-util.js

```diff
@@ -0,0 +1,5 @@
+export function setHash(name) {
+  const shortName = name.substring(0, name.search(/-\d+\s/) + name.match(/-\d+\s/)[0].length).trim();
```

Comment:
This is going to miss the `BIS/TER/...` part at the end of the file.

You're better off doing as the ticket suggests and copying over https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/utils/vr-document-name.js into Valvas so you can just re-use the existing logic.

Alternatively a regex like this should also work: `/(VR \d{4} \d{4} (?:DOC|DEC|MED)\.\d{4}-\d+).*(BIS|TER|QUATER|QUINQUIES|...)/`, but probably better off to just re-use the existing code.